.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -0,0 +1,43 @@
+name: "\U0001F41B Bug Report"
+description: Report an issue or possible bug
+title: "\U0001F41B BUG:"
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ##  Quick Checklist
+        Thank you for taking the time to file a bug report! Please fill out this form as completely as possible.
+
+        ✅ I am using the **latest version of Wrangler**.
```

Comment:
Let's remove this line. It is reasonable that the developers are not necessarily on the latest version and it just adds noise to this template.